src/index.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable no-use-before-define */
+import { displayWeatherInfo } from './functions';
+
+const iconElement = document.querySelector('.weatherIcon');
+const tempElement = document.querySelector('.temperature p');
+const descElement = document.querySelector('.temperatureDescription p');
+const locationElement = document.querySelector('.location p');
+const notificationElement = document.querySelector('.notification');
+const searchInput = document.getElementById('search-input');
+const searchBtn = document.getElementById('search-button');
+
+function setPosition(position) {
+  const { latitude } = position.coords;
+  const { longitude } = position.coords;
+  getWeatherAuto(latitude, longitude);
+}
+
+function showError(error) {
+  notificationElement.style.display = 'block';
+  notificationElement.innerHTML = `<p> ${error.message} </p>`;
+}
+
```

Comment:
Look for a way to separate LOGICAL code from DOM. This file manipulates the API and DOM elements. You could create a file for Weather Api that does calls and pass data to another DOM file, where you do all the displays [ [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) ]

Reply:
Done, thank you.